docs/source/requests.md

```diff
@@ -292,17 +292,17 @@ of a transaction in the Ledger (see [transactions](transactions.md)).
         - `from` (base58-encoded string):
              Identifier (DID) of the transaction author as base58-encoded string
              for 16 or 32 bit DID value.
-             It may differ from `endorser` field who submits the transaction on behalf of `identifier`.
-             If `endorser` is absent, then the author (`identifier`) plays the role of endorser and submits request by his own.
-             It also may differ from `dest` field for some of requests (for example NYM), where `dest` is a 
+             It may differ from the `endorser` field who submits the transaction on behalf of `identifier`.
+             If `endorser` is absent, then the author (`identifier`) plays the role of the endorser and submits a request on his own.
+             It also may differ from `dest` field for some of the requests (for example NYM), where `dest` is a 
              target identifier (for example, a newly created DID identifier).
              
              *Example*:
              
              - `identifier` is a DID of a transaction author who doesn't have write permissions; `endorser` is a DID of a user with Endorser role (that is with write permissions).
              - new NYM creation: `identifier` is a DID of an Endorser creating a new DID, and `dest` is a newly created DID.
         - `reqId` (integer): 
-            Unique ID number of the request with transaction.
+            Unique ID number of the requested transaction.
```

Comment:
I'm a bit confused about this change (and this comment applies to similar change in [other PR](https://github.com/hyperledger/indy-node/pull/1496/files#diff-cb21a9e7664ee87aed23e5c06505b306R135) as well). We have write requests, which are some JSON data sent by clients, and after passing some validation and transformation this data gets written to ledger. Basically write request is an object containing some data, which later will be transformed into transaction - which was reflected in original text, although I agree the it might look weird. On the other hand I might be wrong (I'm not a native english speaker), but your change to "requested transaction" reads like it is all about reading transaction by id, not writing it and assigning unique id. So, what's about changing this to something like this?
```suggestion
            Unique ID number of the request containing transaction to be written.
```

Btw, all other changes look good, thanks for your contribution!

Reply:
Thank you @burdettmary for cleaning up these documents. It is a big help.

The text `Unique ID number of the request with transaction` is awkward English.

I agree with Sergey that we want to be precise in how we talk about the requests and the transactions, but I didn't follow Sergey's explanation. What does the ID apply to?

If it applies to the request, then I like this phrase:

`Unique ID number of the request containing *a* transaction to be written.` (Note the grammar correction.)

If it applies to the transaction within the request, then it should be is simpler to say:

`Unique ID number of the transaction within the request.`

Reply:
@esplinr Thanks for your comments. I believe `Unique ID number of the request containing *a* transaction to be written` is correct since it applies to the request.